README.md

```
@@ -1,102 +1,81 @@
-### Project Demo
+# 4. Collect all images (potato.png, tomato.png, etc.) and put in /images/
+```
+# Run
+```bash
+# Serve locally! (choose one)
+python -m http.server 8000
+# or use VSCode Live Server, or 'http-server' if you use node/npm
+# Open http://localhost:8000/upload.html in your browser
+```
+# Project Documentation
+### Screenshots
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)